### DISCUSSION TITLE
How to now in the app that it was updated?

### DISCUSSION THREAD

**Discussion by timoreichen** [shannah/jdeploy#126] [Q&A]
So the app can show a "That's new"-Button and show the corresponding Releasenotes.

**Comment by shannah** [comment]
You could use the `jdeploy.app.version` system property.  Store somewhere, e.g. in Preferences, on launch.  If it is different than the value you stored last time, then you know it has been updated.

**Comment by timoreichen** [comment]
Cool, thank you!
